rdr_service/dao/study_nph_dao.py

```diff
@@ -216,10 +216,14 @@ def validate(self, order_id: int, nph_participant_id: str, session):
         create_site_exist = self.site_dao.site_exist(session, self.order_cls.createdInfo.site.value)
         collected_site_exist = self.site_dao.site_exist(session, self.order_cls.collectedInfo.site.value)
         finalized_site_exist = self.site_dao.site_exist(session, self.order_cls.finalizedInfo.site.value)
+        client_id = fetch_identifier_value(self.order_cls, "client-id")
+        client_id_exist = self.check_client_id_exist(client_id, session)
```

Comment:
Though I appreciate validation, this check should be removed. The order's table will not have any existing client IDs at first.

Reply:
client_id validation has been removed